website/docs/guides/react.mdx

```diff
@@ -391,7 +391,7 @@ generates:
 >
 > `schema` needs to be your target GraphQL API URL (`"/graphql"` included).
 > `documents` is a glob expression to your `.graphql` files.
-:::
+> :::
```

Comment:
```suggestion
```
can be remove
![image](https://user-images.githubusercontent.com/7361780/162740360-fcdc6613-1e96-4603-8946-c25227844764.png)